node/src/chain_spec.rs

```diff
@@ -32,6 +34,9 @@ pub type ChainSpec = GenericChainSpec<GenesisConfig>;
 const FFNET_CHAIN_SPEC: &[u8] = include_bytes!("./chain_spec/ffnet.json");
 const LATEST_RUNTIME_WASM: &[u8] = include_bytes!("../../runtime/latest.wasm");
 
+/// Chain for local development with an isolated node.
```

Comment:
To keep the terminology consistent with the above:

```suggestion
/// Chain for local development with a single node.
```

Reply:
Fixed